.github/actions/setup-build/action.yml

```diff
@@ -37,11 +43,6 @@ runs:
       python -m pip install -r build-requirements.txt
     shell: bash
 
-  - name: Install prerequisites (Linux)
```

Comment:
Can we keep this in? With this change, the build fails on our fork of torch-mlir which uses standard github runners.

Reply:
Sounds good